src/test/java/uk/gov/hmcts/reform/professionalapi/controller/external/ProfessionalExternalUserControllerTest.java

```diff
@@ -294,24 +300,68 @@ public void test_FindUserStatusByEmail() throws JsonProcessingException {
         newUserResponse.setIdamStatus("ACTIVE");
         ObjectMapper mapper = new ObjectMapper();
         String body = mapper.writeValueAsString(newUserResponse);
+        String email = "test@email.com";
+        RequestContextHolder.setRequestAttributes(new ServletRequestAttributes(httpRequest));
+        when(httpRequest.getHeader(anyString())).thenReturn(email);
+        ResponseEntity<NewUserResponse> responseEntity1 = new ResponseEntity<NewUserResponse>(newUserResponse,
+                HttpStatus.OK);
+
+        when(userProfileFeignClient.getUserProfileByEmail(anyString())).thenReturn(Response.builder()
```

Comment:
please add `verify()` for these two `when().thenReturn()`

Reply:
done